### ISSUE TITLE
[BUG] [Formatter] --reformat not idempotent: {% endblock %} inside a multi-line attribute value with blank_line_after_tag (regression in 1.40.1, not fixed by 1.40.3)

### ISSUE THREAD

**Issue by databyte** [djlint/djLint#2185]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info

- OS: macOS 26.5.1 (arm64)
- Python Version: 3.14.3
- djLint Version: 1.40.3 (regression first appears in 1.40.1)
- template language: django

## Issue

Thanks for 1.40.3. After trying it out, this existing issue surfaced more clearly now that the other issues were resolved.

`djlint --reformat` is **not idempotent** for a `{% block %}…{% endblock %}` that lives inside a multi-line HTML attribute value, when `blank_line_after_tag` includes `endblock`. A single `--reformat` does not reach a fixed point: the closing `" />` is indented one way on the first pass and re-indented on the second pass. So `djlint --check` run immediately after `djlint --reformat` still reports the file needs formatting, which breaks any format-then-verify pre-commit/CI setup.

This looks like the same area fixed by 655afef / #2181–#2183 ("preserve djlint off blocks across tag boundaries"), but a different trigger (`{% block %}` inside an attribute value, no `djlint:off`) that the fix did not cover.

## How To Reproduce

**1. Template** — save as `icon.html` (a real SVG icon base template; the `{% block %}` is embedded in the multi-line `d="…"` attribute):

```html
<svg viewBox="0 0 24 24">
  <path stroke-linecap="round" stroke-linejoin="round" d="
  {% block svg_path %}
  {% endblock svg_path %}

  " />
</svg>
```

**2. Config** — the trigger requires `blank_line_after_tag` to include `endblock`. Save as `pyproject.toml` next to the template:

```toml
[tool.djlint]
indent = 2
profile = "django"
blank_line_after_tag = "endblock"
```

(Equivalently, pass `--blank-line-after-tag endblock --profile django --indent 2` on the command line.)

**3. Run `--reformat` twice — the file changes on both runs:**

```bash
djlint icon.html --reformat        # 1st pass: closing tag becomes "    " /> (4 spaces)
djlint icon.html --reformat        # 2nd pass: closing tag becomes "   " />  (3 spaces)  ← should have been a no-op
```

Diff produced by the **second** `--reformat` (proving the first pass was not a fixed point):

```diff
-    " />
+   " />
```

**4. Or the CI-facing symptom — `--reformat` then `--check`:**

```bash
djlint icon.html --reformat
djlint icon.html --check
```

**Actual output:**
```
1 file would be updated.
```

**Expected output:** `0 files would be updated.` — a single `--reformat` should already be a fixed point.

## Version bisect

| Version | Result |
|---|---|
| ≤ 1.40.0 | ✅ stable (single `--reformat` is a fixed point) |
| **1.40.1** | ❌ non-idempotent (introduced here) |
| 1.40.2 | ❌ non-idempotent |
| **1.40.3** | ❌ still non-idempotent (the #2181–#2183 fix did not cover this case) |

```bash
uvx djlint@1.40.0 icon.html --reformat && uvx djlint@1.40.0 icon.html --check   # 0 files would be updated
uvx djlint@1.40.3 icon.html --reformat && uvx djlint@1.40.3 icon.html --check   # 1 file would be u
```

**Comment by databyte** [comment]
Working on the PR now.

**Comment by databyte** [comment]
PR is up
